web-app/resources/java-worker/doc/wps-acorn.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<wps:Execute service="WPS" version="1.0.0"
+  xmlns:wps="http://www.opengis.net/wps/1.0.0" xmlns:ows="http://www.opengis.net/ows/1.1"
+  xmlns:ogc="http://www.opengis.net/ogc" xmlns:xlink="http://www.w3.org/1999/xlink"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.opengis.net/wps/1.0.0 http://schemas.opengis.net/wps/1.0.0/wpsExecute_request.xsd">
+
+
+  <ows:Identifier>gs:GoGoDuck</ows:Identifier>
+  <wps:DataInputs>
+    <wps:Input>
+      <ows:Identifier>layer</ows:Identifier>
```

Comment:
Running as WPS (curl in the README.md) does not work. Even after changing the xml to `<ows:Identifier>layerName</ows:Identifier>` the curl output still complains about `layerName` being null
??